packages/nextjs-template/components/DendronSearch.tsx

```diff
@@ -255,13 +265,13 @@ function DendronSearchComponent(props: DendronCommonProps & SearchProps) {
       value={searchQueryValue}
       getPopupContainer={(trigger) => trigger.parentElement}
       // @ts-ignore
-      onClick={results === SearchMode.SEARCH_MODE ? () => null : onClickLookup}
+      onClick={searchMode === SearchMode.SEARCH ? () => null : onClickLookup}
```

Comment:
I am wondering what the `onClick` actually do. Quickly playing on site it does nothing or rather on the first click it opens and suddenly closes again.

Reply:
onclick, when the mode is Lookup(present behavior) fetches notes and updates the autocomplete with the results somewhat similar to how lookup behaves when no note is open in local workspace. The behavior is sketchy sometimes, I can take a look at this in a followup PR